chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ActivityController.java

```diff
@@ -847,26 +928,34 @@ public ActivityMetadataPage metadataForPendingSurveillances(@RequestParam(requir
 
     @Deprecated
     @Operation(summary = "DEPRECATED. Get auditable data for a specific certification body.",
-    description = "A start and end date may optionally be provided to limit activity results.  "
-            + "Security Restrictions: ROLE_ADMIN and ROLE_ONC may see activity for all certification bodies.  "
-            + "ROLE_ACB can see their own information.",
+            description = "A start and end date may optionally be provided to limit activity results.  "
+                    + "Security Restrictions: ROLE_ADMIN and ROLE_ONC may see activity for all certification bodies.  "
+                    + "ROLE_ACB can see their own information.",
             deprecated = true,
-            security = { @SecurityRequirement(name = SwaggerSecurityRequirement.API_KEY),
-                    @SecurityRequirement(name = SwaggerSecurityRequirement.BEARER) })
+            security = {
+                    @SecurityRequirement(name = SwaggerSecurityRequirement.API_KEY),
+                    @SecurityRequirement(name = SwaggerSecurityRequirement.BEARER)
+            })
     @RequestMapping(value = "/acbs/{id}", method = RequestMethod.GET, produces = "application/json; charset=utf-8")
 
     public List<ActivityDetails> activityForACBById(@PathVariable("id") final Long id,
             @RequestParam(required = false) final Long start, @RequestParam(required = false) final Long end)
-                    throws JsonParseException, IOException, EntityRetrievalException, ValidationException {
-        CertificationBodyDTO acb = resourcePermissions.getAcbIfPermissionById(id); // throws 404 if ACB doesn't exist
+            throws JsonParseException, IOException, EntityRetrievalException, ValidationException {
+        CertificationBodyDTO acb = resourcePermissions.getAcbIfPermissionById(id); // throws
```

Comment:
I don't know what setting you have that does things like this to these comments, but maybe consider turning it off? Or updating it to not have changed this? There are a lot of changes to comments in this file just to change spacing and line breaks, and it makes it really hard to figure out what actually changed

Reply:
I've tried to turn this off...  It only seems to happen to the chpl-api project and not chplk-service.  Made a few morte changes, and we'll see what happens.